docs/operations/cx-server-operations-guide.md

```diff
@@ -181,16 +203,18 @@ You can do it by executing below commands.
 ```
 
 #### Plugins
-All the plugins that are required to run the SAP S/4HANA Cloud SDK Continuous Delivery Pipeline and the Piper steps
-are already pre-installed. If you update or downgrade them to a specific version, it will be lost every time the `cx-server` image is updated. 
+
+All the plugins that are required to run the SAP Cloud SDK Continuous Delivery Pipeline and the Piper steps are already pre-installed.
+If you update or downgrade them to a specific version, it will be lost every time the `cx-server` image is updated. 
 All the plugins are updated with the latest version. 
 If there is a need, the user can install additional plugins and configure them. 
 However, the `cx-server update` will not update the plugins that are custom installed.
 
 #### Troubleshooting
  
 ##### Disk space cleanup
-If you encounter an issue related to diskspace on a cx-server, you can free up space by launching [system prune](https://docs.docker.com/engine/reference/commandline/system_prune/) command.
+
+If you encounter an issue related to disk space on a cx-server, you can free up space by launching [system prune](https://docs.docker.com/engine/reference/commandline/system_prune/) command.
```

Comment:
Does not related to the topic of the PR (Document release process)  - like also all the newlines. Blurs the focus of the PR and makes it harder to read.

Reply:
Strictly speaking I agree, but I don't want to revert those changes because they make the document easier to read and edit. I root for pragmatic choice here.

Reply:
The more focused and small a PR is the more I like it to review. This means: the more focused and small I pull request is the more likely it is to get an approval. I mean here it is rather simple. 

[nit] It gets hairy in case there is a pull request dealing with several dimensions at the same time. Than you agree on one thing, but maybe not on another. The aggreed feature has to wait, despite it can be in general merged and so on, and so on ...